main/inc/lib/sessionmanager.lib.php

```diff
@@ -5429,4 +5428,266 @@ public static function getCoursesListByCourseCoach($coachId)
         ));
     }
 
+    /**
+     * Returns the list of session (name, short description, start date, end date) from category.
+     * The short description is an extra field value
+     * @param int $categoryId
+     * @param string $target
+     * @return mixed
+     */
+    public static function getSessionBriefListByCategory($categoryId, $target) {
```

Comment:
This method also returns extra fields, which is confusing because the name of the method says "brief", while another method further down says "AndExtra" !
Please clarify the methods names or the methods code.